### PR TITLE
document pre-docker build command

### LINKED ISSUE #2163: [BUG] Docker build fails
# Bug Report

## Context

Build from Dockerfile fails with error on step 42

### How do we replicate the issue?
Using a clean checkout from git and the master branch, run `docker build --no-cache .`
The error also occurs with v10.1.2

### Actual Behavior
```
Step 42/72 : COPY   ./target/bin/*   ./target/scripts/*.sh   ./target/scripts/startup/*.sh   ./target/docker-configomat/configomat.sh   /usr/local/bin/
COPY failed: file not found in build context or excluded by .dockerignore: stat target/docker-configomat/configomat.sh: file does not exist
```
No errors or warnings are reported prior to the message above.

### Expected Behavior
Successful build

## Your Environment
Docker version 20.10.7, build f0df350

### PR DESCRIPTION
**Pull request by michaeljensen**
# Description

Document steps for building your own docker image

Fixes #2163

## Type of change

- [x] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (README.md or the documentation under `docs/`)
- [x] If necessary I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by wernerfred** [comment]
Didn't you think about introducing a new page? https://github.com/docker-mailserver/docker-mailserver/issues/2163#issuecomment-910810314

**Review by NorseGaud — CHANGES_REQUESTED** [review]
Can you move this into a docs page? Having it in the readme suggests that people should build themselves. I don't think we should be suggesting that due to all of the potential problems experienced on build.

**Comment by michaeljensen** [comment]
Good point, I didn't think about what it would imply to have it in the readme.  I'll move it over to a docs page.

File: README.md
Comment (polarathene) [inline README.md:158]: Title is a bit vague.

- Building own custom Dockerfile or the one in git?
- Building own Docker image could also extend ours as a base image

Perhaps "Building the project Dockerfile from git" or similar? (maybe source instead of git)

In my unpublished docs it was for devs/contributors to build as part of testing their PR locally, but for users I think they can have different intentions for their own docker image.

Comment (casperklein) [reply README.md:158]: IMO the title is fine and I don't think, that there will be misunderstandings. Building docker images also presupposes a basic knowledge about it.

BTW, to make it even simpler: Instead of "run `git submodules..` in your previous cloned repo" etc., maybe we should just advice to `git clone --recurse-submodules https://github.com/docker-mailserver/docker-mailserver`.

1. Clone repo (recursively) --> `git clone..`
2. Build image --> `docker build -t  my-DMS-build .`

**Comment by georglauterbach** [comment]
Can this PR be finalised?

**Comment by michaeljensen** [comment]
I had a few things come up that required my attention and pulled me away from this.  I'll take a bit of time and get it finished up today.

**Comment by michaeljensen** [comment]
I've moved it over to tutorials.
I also included the original error message I was seeing so that hopefully others can find if when searching for the error message text.

**Review by wernerfred — APPROVED** [review]
LGTM 👍🏻  
thanks

**Review by georglauterbach — APPROVED** [review]

**Review by casperklein — APPROVED** [review]

**Comment by georglauterbach** [comment]
@NorseGaud I hope this is fine with youu, but I removed you review to merge this PR (because your review was blocking this PR but it was also outdated). Since we have 3 approvals, I figured this is fine:) I will correct anything if you still want to see some changes:)

**Pull request merged**
